### ISSUE TITLE
News items not shown and after restart incorrect collumns

### ISSUE THREAD

**Issue by WSpu** [QuiteRSS/quiterss#800]
Sometimes QuiteRss doesn't show any news anymore when I select a feed. I tried changing the filter or to search for some news but no news items are shown and therefore the browser also doesn't show the content of the news item. When I exit and restart QuiteRss the news items are shown again but there are so many collumns that the content cannot be shown correctly.
![quiterss_no_news](https://cloud.githubusercontent.com/assets/13487579/8881171/19ead5f6-323e-11e5-8236-16dcaf5d3766.png)
When I restart QuiteRss again all is normal again.

If I remember correctly the problem with no news items was also there in version 0.18.1 but the problem with the many new collums is new to version 0.18.2

**Comment by WSpu** [comment]
Problem still occurs frequently and today I noticed that the 'Incomming News' notification also was empty. After restarting QuiteRSS I got the same result as the picture above. After closing and starting QuiteRSS for the second time I could use the program again.

![image](https://cloud.githubusercontent.com/assets/13487579/10948713/4808a5c0-832f-11e5-9178-50da352573bf.png)

Is there any hope for a solution for this issue?
